website/videoBlog/2023-12-11-Simplifying-Big-Data-Setting-Up-SparkSQL-Hive-Thrift-Server-and-Hudi-with-Beeline-in-Minutes.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: "Simplifying Big Data: Setting Up Spark SQL, Hive Thrift Server, and Hudi with Beeline in Minutes"
+last_modified_at: 2023-12-20T16:54:38.964863-07:00
+authors:
+- name: Soumil Shah
+category: blog
+image: /assets/images/video_blogs/2023-12-11-Simplifying-Big-Data-Setting-Up-SparkSQL-Hive-Thrift-Server-and-Hudi-with-Beeline-in-Minutes.png
+navigate: "https://www.youtube.com/watch?v=lCorHcx2mvc"
+tags:
+- guide
+- aws
```

Comment:
avoid plain `aws` tag and go with fully qualified tags like `aws emr` for example